site/docs/v1/tech/email-templates/templates-replacement-variables.adoc

```diff
@@ -698,8 +696,15 @@ ${code}
 === Replacement Variables
 
 [.api]
-[field]#application# [type]#[Application]# [since]#Available since 1.28.0#::
+[field]#application# [type]#[Application]# [since]#Available since 1.46.0#::
 The Application object, see the link:/docs/v1/tech/apis/applications[Application API] for field definitions.
++
+*Note*:
+This object may not be available depending upon when this template is constructed. If you utilize this object in your template, ensure you first check to see if it is defined. You can check for this variable safely in freemarker by wrapping the variable as such: `${(application)!""}`. This object is not available on the email template when:
```

Comment:
If we want to help them understand when the application is available, we should probably show them an if statement. All this will do is print the entire object in JSON or print empty string.